.github/workflows/ci.yml

```diff
@@ -4,12 +4,12 @@
 #
 # This workflow will install a prebuilt Ruby version, install dependencies, and
 # run tests and linters.
+
+
 name: "Ruby on Rails CI"
 on:
   push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+    branches: [ '*' ]
```

Comment:
Why?

Reply:
@micorix because we want to run this workflow on every commit. The previous config doesn't run this workflow on pull requests to pull requests and drafts

Reply:
We can enable CI on more branches if you insist, but I'd rather not do that. Here's why:
1. Does drafts need CI? Besides r u sure draft PRs with master branch as merge target don't trigger workflows?
2. I'm not sure if making PRs to branches with already-open PR to master is such a good thing, coz
  * It may lead to duplicated code reviews 
  * and it encourages contributors to make large PRs
  * not sure if i've seen sth like it before (altho it's not 100% proper argument against it)

Which leads to another thing. I've seen that your PRs contain lots of changes (i.e. this one: +536 −93 LoC). I'm not a backend dev but I think there should be some way to divide huge PRs into lots of smaller ones. That way code review (probably) may be done quicker and development would be more rapid. (Altho we still need to keep in mind assumption that master should be deployable at any time)

Reply:
1 Yes you want to see if tests pass from that commit and if linting is ok.
2 Because we have one feature branch at a time the pull request is quite large and some things are done in branches where it doesn't make sense to branch out of master and that smaller thing is finished before the big thing is finished so this smaller thing is merged into the bigger merge. 

You don't have to worry about there being a lot of changes in a single pr because a feature that is being developed is quite robust and contains a lot of autogenerated code related to rswag and a lot of line changes in Gemfile.lock, Gemfile etc. that are ignored during code review. I was doing code review with Wojtek yesterday and it wasn't a problem.

Reply:
Yeah, it'd be way better to do smaller PRs and to diverge them from the master, especially when working on independent features - like here, at the very least, swagger could've been a separate PR. It'd make the process way faster, for sure. But what we're doing right now is ok for me for now - I hope once @Anakin100100 gets a little bit more experience with Git, those PRs will start to look better ;)
As for this change, while I do agree we should strive not to make PRs to other PRs, I don't see many disadvantages of not running CI for all branches, and I could see why we would want to have such PRs (e.g., if we ware to use gitflow).